index.js

```diff
@@ -1 +1,2 @@
 module.exports = require('./lib/quibble')
+x``
```

Comment:
Forgive me for not understanding, but what's this change do?

Reply:
It is nothing. Accidentally added. Now removed. THanks!